org.eclipse.xtext.buildship/src/org/eclipse/xtext/buildship/IgnoreBuildDirContribution.java

```diff
@@ -7,35 +7,44 @@
  *******************************************************************************/
 package org.eclipse.xtext.buildship;
 
+import java.io.File;
 import java.util.Collections;
+import java.util.Optional;
 
-import org.eclipse.buildship.core.internal.CorePlugin;
-import org.eclipse.buildship.core.internal.preferences.PersistentModel;
+import org.eclipse.buildship.core.GradleBuild;
```

Comment:
we should also make sure we use buildship 3.x in the manifest
and also do the same pom thing we do for codemining
(have to find out which release has buildship 3.x)

Reply:
Done